### ISSUE TITLE
Usage for fs-fuse

### ISSUE THREAD

**Issue by happybeing** [piranna/fs-fuse#3]
Hi,

Do you have some information on how to use fs-fuse, or an example etc?

I'm looking at providing a FUSE mountable filesystem for SAFE Network, and wondering if a good way to do it would be to first implement this as an fs object, then use fs-fuse to enable it to be mounted. Killing two birds with one stone.

Thanks

**Comment by piranna** [comment]
> wondering if a good way to do it would be to first implement this as an fs object, then use fs-fuse to enable it to be mounted

Yeah, that's exactly the use case of `fs-fuse`, you develop your filesystem as a `fs`-like module that others can use, and `fs-fuse` will allow you to mount it as a reguar filesystem. Do you have any link about the SAFE Network? Seems interesting...

**Comment by happybeing** [comment]
Having had a brief look at `fs` and `FUSE` I'm now thinking it will be easier to start by going straight to `FUSE` although I would like to have an `fs-safe` for obvious reasons, but maybe later.

I'm very interested in SAFE Network, did a presentation at the recent DevCon on hooking it up with Tim Berners-Lee's Solid which went down really well. It's an awesome project. This [10 key facts about SAFE Network](https://safenetforum.org/t/10-key-facts-about-safe-network/12948?u=happybeing) is still my favourite intro link, so try that and feel free to ask me stuff directly (I'm `@happybeing` over there, also https://twitter.com/safepress)

Do you have a quick intro to Node OS, that looks interesting too?! I see http://node-os.com/ but am not sure what it adds up to.
